apps/developer-portal/src/components/applications/help-panel/samples-guide.tsx

```diff
@@ -0,0 +1,225 @@
+                .then((response) => {
+                    protocolConfigs = {
+                        ...protocolConfigs,
+                        [protocolName]: response
+                    };
+
+                    selectedProtocolList.push(protocolName);
+                })
+                .catch((error) => {
+                    if (error?.response?.status === 404) {
+                        return;
+                    }
+
+                    if (error?.response && error?.response?.data && error?.response?.data?.description) {
+                        dispatch(addAlert({
+                            description: error.response?.data?.description,
+                            level: AlertLevels.ERROR,
+                            message: t("devPortal:components.applications.notifications.getInboundProtocolConfig" +
+                                ".error.message")
+                        }));
+
+                        return;
+                    }
+
+                    dispatch(addAlert({
+                        description: t("devPortal:components.applications.notifications.getInboundProtocolConfig" +
+                            ".genericError.description"),
+                        level: AlertLevels.ERROR,
+                        message: t("devPortal:components.applications.notifications.getInboundProtocolConfig" +
+                            ".genericError.message")
+                    }));
+                })
+                .finally(() => {
+                    setInboundProtocolConfigs(protocolConfigs);
+                });
+        });
+    },  []);
+
+    useEffect(() => {
+        if (inboundProtocolConfigs == undefined) {
+            return;
+        }
+
+        const configs = {
+            clientHost: deploymentConfigs.clientHost,
+            clientID: inboundProtocolConfigs.oidc.clientId,
+            loginCallbackURL: inboundProtocolConfigs.oidc.callbackURLs[0],
+            logoutCallbackURL: inboundProtocolConfigs.oidc.callbackURLs[0],
+            serverOrigin: deploymentConfigs.serverOrigin,
+            tenant: deploymentConfigs.tenant,
+            tenantPath: deploymentConfigs.tenantPath
+        };
+
+        setAuthConfig(configs);
+    }, [ inboundProtocolConfigs ]);
+
+    /**
+     * Todo Remove this mapping and fix the backend.
+     */
+    const mapProtocolTypeToName = ((type: string): string => {
+        let protocolName = type;
+        if (protocolName === "oauth2") {
+            protocolName = SupportedAuthProtocolTypes.OIDC;
+        } else if (protocolName === "passivests") {
+            protocolName = SupportedAuthProtocolTypes.WS_FEDERATION;
+        } else if (protocolName === "wstrust") {
+            protocolName = SupportedAuthProtocolTypes.WS_TRUST;
+        } else if (protocolName === "samlsso") {
+            protocolName = SupportedAuthProtocolTypes.SAML;
+        }
+
+        return protocolName;
+    });
+
+    const downloadConfigFile = () => {
```

Comment:
Add return as void if nothing is returned.

Reply:
Resolved with fb5681d